src/css/controls/flags/ads.less

```diff
@@ -87,15 +87,6 @@
             }
         }
     }
-
-    .jw-nextup-container {
```

Comment:
why would next up show during ads ? Is this for banner ads ?

Reply:
If it was `mobile-touch-target` isn't tall enough